roadmaps/2022.md

```diff
@@ -91,33 +91,73 @@ The core team has two main goals for this quarter: to improve the product and al
+
+## Q4 2022
+
+This quarter is the last one we have to prepare v1 in good condition. This way, we have prioritized tasks that would help the team deliver Meilisearch v1: a stable and impactful version of Meilisearch.
+It also means we will de-prioritize implementations that could impact in a bad way the critical features of Meilisearch, like improving the indexing speed in a way that would risk making the indexing fail.
+
+Q4 is also the quarter where the Hacktoberfest is happening and this will lead us to spend time on it. We created a lot if issues during Q3, especially regarding the language support, and we hope attract the community and make them contribute in October.
+
+### v1 work
+
+- [ ] Be production ready for v1
+  - [ ] Release the missing production-ready features: index swap, cancel/delete of tasks, and other missing features noticed by the product team.
+  - [ ] Ensure we don't have any bugs/crashes impacting the stability of Meilisearch. The RAM consumption should be appropriate when indexing and searching, or at least we must be transparent with the users about it.
+  - [ ] Define more precisely what kind of distribution we support. Transparency is the key to this point. Then, we need to ensure we have tests/CIs appropriate to them.
+  For example, what about the Debian 10 distribution? And what about Windows and the prioritization we should give to it? Indeed, all the features of Meilisearch do not work with Windows so far, like dumps. It's important to provide transparency about it.
+- [ ] Define our versioning policy process post v1
+- [ ] Define our beta/deprecated process post v1
+- [ ] Create some benchmarks of concrete use cases. We need to define the real objectives of the project, how to ensure to make it useful at the end. Create a Product Squad for it and manage it as a new feature of Meilisearch could be a good start.
+
+### Team
+
+- [ ] Onboard the new senior software engineer: at the beginning of November, we welcome a new software engineer to the team.
+- [ ] This is the end of the internship of one of our teammates. Following a promotion, we want to integrate him in the sprint as any developer of the team.
+- [ ] Increase the communication and synchronization
+  - [ ] Reconsider some of our meeting formats: weekly meetings and retrospectives.
+  - [ ] Define some communication rules to ensure everyone gets all the information and is reactive and accountable when pinged (GitHub and Slack).
+- [ ] Reconsider the name of the team that is currently "core team". We noticed this name is not adapted to the job of the team; indeed "core team" in most companies is often the central team that makes important decisions. Also, this name can feel uninclusive, giving the team the feeling of being central and more important than the others. Even if we don't have the time to find a new name, we would like to, at least, start the discussions with the whole team during Q4.
```

Comment:
`if we don't have the time to find a new name` I feel that this subject is an important one, and I'd prefer to take all the time needed to choose a new one. Or, if it's not possible, I'd prefer to delay the whole subject to the next quarter. wdt?

Reply:
I agree with you, I did not want to impose any priority since we did not decide it together yet. If this is something you all @meilisearch/core-team want to prioritize as well, I'm in!

Reply:
"Engine Team" 🔥 😝

Reply:
teameili